src/core/colorspace.js

```diff
@@ -1057,6 +1057,9 @@ const CalRGBCS = (function CalRGBCSClosure() {
     if (color <= 0.0031308) {
       return adjustToRange(0, 1, 12.92 * color);
     }
+    if (color >= 0.99554525) {
```

Comment:
Please add a comment above this line, explaining exactly how this value was determined!
As-is, this line looks quite wrong until you've actually read https://github.com/mozilla/pdf.js/pull/11963#issue-426882562